app/src/app/[lng]/auth/forgot-password/page.tsx

```diff
@@ -71,9 +71,8 @@ export default function ForgotPassword(props: {
             {t("reset-password")}
           </Button>
           <Button
-            type="reset"
```

Comment:
Why did you remove the type property? I think that might prevent this button from working correctly?

Reply:
because the property was duplicated